GitUI/CommandsDialogs/FormDeleteRemoteBranch.resx

```diff
@@ -118,7 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="labelDeleteBranchWarning.Text" xml:space="preserve">
-    <value>When you delete a branch the commits can get lost because nothing points to them.
-Make sure that commits under remote branch(-es) to be deleted are reachable before proceeding.</value>
+    <value>The commits referenced by the deleted branch will be lost
+if no other branch or tag references them. Make sure there
+is a branch or tag referencing the commits you don't want to loose.
+If you accidentaly deleted a wrong branch, go to the Commands | Show reflog form
+in order to recover the lost commits.</value>
```

Comment:
Few typos.
Here's my corrected version:

```
The commits referenced by the deleted branch will be lost
if no other branch or tag reference them. Make sure there
is a branch or tag referencing the commits you wish to keep.
If you have accidentally deleted a wrong branch, go to the Commands | Show reflog form
in order to recover the lost commits.
```

I'm split on `no other branch or tag references them.` I think it should say `reference`...
@vbjay are you a native speaker to tell the right one?

Reply:
One of my native speaking colleagues reckons it has to be "references", so let's go with that.

```
The commits referenced by the deleted branch will be lost
if no other branch or tag references them. Make sure there
is a branch or tag referencing the commits you wish to keep.
If you have accidentally deleted the wrong branch, go to Commands | Show reflog form
in order to recover the lost commits.
```

Reply:
Why have you also changed 
`deleted a wrong branch` to `deleted the wrong branch`? It is contrary to my understanding of the language.